Dockerfile

```diff
@@ -9,7 +9,7 @@ ADD . /gopath/src/github.com/drone/drone/
 WORKDIR /gopath/src/github.com/drone/drone
 
 RUN apt-get update
-RUN apt-get -y install zip libsqlite3-dev sqlite3 1> /dev/null 2> /dev/null
+RUN apt-get -y install zip libsqlite3-dev sqlite3 >/dev/null 2>&1
```

Comment:
I think the real problem is that google/golang image is based on debian and it uses dash (not bash) as primary shell, so `&>/dev/null` won't work.